cmd/security-agent/app/config.go

```diff
@@ -0,0 +1,80 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build kubeapiserver
+
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	SecurityAgentCmd.AddCommand(configCommand)
+}
+
+var configCommand = &cobra.Command{
+	Use:   "config",
+	Short: "Print the runtime configuration of a running security agent",
+	Long:  ``,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		if flagNoColor {
+			color.NoColor = true
+		}
+
+		// we'll search for a config file named `datadog.yaml`
+		config.Datadog.SetConfigName("datadog")
+		err := common.SetupConfig(confPath)
+		if err != nil {
+			return fmt.Errorf("unable to set up global security agent configuration: %v", err)
+		}
+
+		err = config.SetupLogger(loggerName, config.GetEnv("DD_LOG_LEVEL", "off"), "", "", false, true, false)
+		if err != nil {
+			fmt.Printf("Cannot setup logger, exiting: %v\n", err)
+			return err
+		}
+
+		err = util.SetAuthToken()
+		if err != nil {
+			return err
+		}
+
+		runtimeConfig, err := requestConfig()
+		if err != nil {
+			return err
+		}
+
+		fmt.Println(runtimeConfig)
+		return nil
+	},
+}
+
+func requestConfig() (string, error) {
+	c := util.GetClient(false)
+	apiConfigURL := fmt.Sprintf("https://localhost:%v/config", config.Datadog.GetInt("security_agent.cmd_port"))
```

Comment:
It should be `https://localhost:%v/agent/config` ? otherwise it doesn't seem to work

Reply:
You are right, fixing.